### PR TITLE
Implement load scenario with cloning project

### PR DESCRIPTION
**Pull request by musienko-maxim**
* Extend performance Jmeter scenario actions that emulate a cloning maven project into a workspace
* Change description and add info with scenarios
  **Depended issue:**  https://issues.jboss.org/browse/CRW-145
  **Note:** all scenarios have been checked on the rial cluster: `https://console.iokhrime-ocp.crew`
  @Ohrimenko1988 @rhopp @dmytro-ndp - rview pls.

### CODE REVIEW

**Comment by SkorikSergey** [comment]
Can one of the admins verify this patch?

**Comment by musienko-maxim** [comment]
ci-build

File: test/codeready-test-performance/src/test/jmeter/codeready-test-scenario-cloning-project.jmx [hunk 1]
```diff
@@ -0,0 +1,1959 @@
...
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="crw.loopCount" elementType="Argument">
+            <stringProp name="Argument.name">crw.loopCount</stringProp>
+            <stringProp name="Argument.value">1</stringProp>
+            <stringProp name="Argument.metadata">=</stringProp>
+          </elementProp>
+          <elementProp name="git.url" elementType="Argument">
+            <stringProp name="Argument.name">git.url</stringProp>
+            <stringProp name="Argument.value">${__P(git.url,https://github.com/musienko-maxim/web-java-spring.git)}</stringProp>
```
Comment (dmytro-ndp) [inline test/codeready-test-performance/src/test/jmeter/codeready-test-scenario-cloning-project.jmx]: IMHO, public account https://github.com/che-samples/web-java-spring.git is more suitable here.

File: test/codeready-test-performance/README.adoc [hunk 2]
```diff
@@ -38,4 +38,13 @@ In the **codeready-test-scenario-with-opening-file.jmx** file implemented next s
...
\ No newline at end of file
+* In the end - performs clean up actions in the separated Threadgroup, remove all users
+------------------
+In the **codeready-test-scenario-cloning-project.jmx** file implemented next scenario:
+* Creates virtual user(s)=amount of the treads (-Dtest.threads property)
+* Each virtual user login and create a workspace
+* Each virtual user waits workspace creation (RUNNING) status
+* Each user run command like git clone git.url
+* Waiting exec 0 status from EXEC agent (this is mean succesfull cloning)
+* After cloning by user(s), the workspace is stoped  and removed
```
Comment (dmytro-ndp) [inline test/codeready-test-performance/README.adoc]: typo stoped > stopped

File: test/codeready-test-performance/README.adoc [hunk 3]
```diff
@@ -38,4 +38,13 @@ In the **codeready-test-scenario-with-opening-file.jmx** file implemented next s
 * Open the pom.xml file from the current project
 * Check that file has been opened properly (check expected content in the responce)
 * After creation the virtual user(s) stop the workspace and remove the workspace
-* In the end - performs clean up actions in the separated Threadgroup, remove all users
\ No newline at end of file
+* In the end - performs clean up actions in the separated Threadgroup, remove all users
+------------------
+In the **codeready-test-scenario-cloning-project.jmx** file implemented next scenario:
+* Creates virtual user(s)=amount of the treads (-Dtest.threads property)
+* Each virtual user login and create a workspace
```
Comment (dmytro-ndp) [inline test/codeready-test-performance/README.adoc]: login > logins, create > creates

File: test/codeready-test-performance/README.adoc [hunk 4]
```diff
@@ -38,4 +38,13 @@ In the **codeready-test-scenario-with-opening-file.jmx** file implemented next s
...
+* Each virtual user waits workspace creation (RUNNING) status
+* Each user run command like git clone git.url
```
Comment (dmytro-ndp) [inline test/codeready-test-performance/README.adoc]: run > runs

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
Results of automated E2E tests of Codeready on OCP:
[Build details](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/codeready-pr-test/39/)
[Test report](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/codeready-pr-test/39//E2E_20tests_20report/)

**Comment by SkorikSergey** [comment]
ci-buld

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
Results of automated E2E tests of Codeready on OCP:
[Build details](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/codeready-pr-test/46/)
[Test report](https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/codeready-pr-test/46//E2E_20tests_20report/)
docker image: quay.io/crw_pr/server:pr106

**Comment by nickboldt** [comment]
Please update this and merge to 6.19.x and master branches, where appropriate. 6.18.x is dead.

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
testPR

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
PR needs to be updated to 6.19.x, and shouldn't contain changes which are not related to performance testing.

**Comment by SkorikSergey** [comment]
ci-test

**Comment by kpiwko** [comment]
So this was closed and replaced with other PR?

**Comment by musienko-maxim** [comment]
Yes i am going to improve the scripts (add some stabilize parts and add possibility to save info about min., max., and average time for cloning, opening file and build operations)

**Comment by kpiwko** [comment]
thanks @musienko-maxim

**Pull request closed without merging**